CHANGELOG.md

```diff
@@ -0,0 +1,59 @@
+# CHANGELOG
+
+All notable changes to this project will be documented in this file.
+
+## Unreleased
```

Comment:
There are a bunch of things that are unreleased 0.3 features that you haven't listed. Are we going to add these in another pull request?

Will each pull request be responsible for updating this file if it is a significant change, or will we do one PR for each release that updates this file en masse?

Do entries here correlate 1:1 with issues/PRs?

Reply:
These are interesting questions. I think, worthy of discussion in SIG meeting. Added the agenda for upcoming meeting.

Reply:
Per SIG. This is good as-is. Thanks mayur